storage/src/mmr/mem.rs

```diff
@@ -12,6 +14,10 @@ pub struct Mmr<H: CHasher> {
     // The nodes of the MMR, laid out according to a post-order traversal of the MMR trees, starting
     // from the from tallest tree to shortest.
     nodes: Vec<Digest>,
+    // The position of the very oldest element still maintained by the MMR. Will be 0 unless
+    // forgetting has been invoked. If non-zero, then proofs can only be generated for elements with
+    // positions strictly after this point.
```

Comment:
Is it "after" or "including and after"?

Reply:
The former, which is why I say "strictly after". LMK if you think I should make this more clear.